multiregion/docs/multiregion.rst

```diff
@@ -447,11 +534,87 @@ There is a script you can run to collect the JMX metrics from the command line,
       multi-region-sync: 4
       multi-region-async: 4
       multi-region-default: 4
+      under-min-isr-promotion: 4
+      under-replicated-promotion: 4
+      leader-is-observer-promotion: 4
 
 
 Failover and Failback
 ---------------------
 
+Degrade Region
+~~~~~~~~~~~~~~
+
+In this section, you will simulate a broker failure in the ``west`` region.
+
+#. Run the following command to stop the Docker containers corresponding to the ``west`` region:
```

Comment:
This statement needs tweaking, not all west region was taken down

Reply:
good point, I changed to to say that we are killing one broker container in west.